### PR TITLE
Added sample to delete stream data

### PR DESCRIPTION
**Pull request by liskaj**

### CODE REVIEW

File: streams/delete-stream-parameter-data.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const schema = await client.getModelSchema(defaultModel.modelId);
+    // STEP 4 - get streams and find stream by name
+    const streams = await client.getStreams(defaultModel.modelId);
+    const stream = streams.find(s => s[QC.Name] === STREAM_NAME);
+
+    if (!stream) {
+        throw new Error(`Stream ${STREAM_NAME} not found`);
+    }
+    // STEP 5 - get stream data
+    const streamKey = Encoding.toFullKey(stream[QC.Key], true);
```
Comment (bakharal) [inline streams/delete-stream-parameter-data.js:40]: There is no need to pass in a flag here. API knows the exact GUID context and can add a flag automatically.

**Review by bakharal — CHANGES_REQUESTED** [review]

File: streams/delete-stream-parameter-data.js (hunk 1 above)
Comment (liskaj) [reply streams/delete-stream-parameter-data.js:40]: I tried this with short key but I got an error (400 - Bad Request) when calling GET timeseries/models/:urn/streams/:streamKey. However it works fine with full key.

Comment (liskaj) [reply streams/delete-stream-parameter-data.js:40]: POST timeseries/models/:modelId/deletestreamsdata also doesn't work with short keys. Did I miss something, pls?

Comment (bakharal) [reply streams/delete-stream-parameter-data.js:40]: I see. It's a bug, Let's hold this PR and fix the API then.

**Review by bakharal — APPROVED** [review]

**Pull request merged**